test/test_alexa_app_unknown_type_request.js

```diff
@@ -21,25 +21,74 @@ describe("Alexa", function() {
 
         var mockRequest = mockHelper.load("unknown_type_request.json");
 
-        it("invokes a globally defined error function", function() {
+        it("invokes a globally defined error function that throws an error", function() {
           testApp.error = function(e, request, response) {
```

Comment:
Now create a new testApp instance and setting the error handler before each test.